core/controllers/editor.py

```diff
@@ -583,10 +573,11 @@ def put(self, exploration_id):
             'email_preferences': exploration_email_preferences.to_dict()
         })
 
+
 class ResolvedAnswersHandler(EditorHandler):
     """Allows learners' answers for a state to be marked as resolved."""
 
-    @require_editor
+    @acl_decorators.can_edit_exploration
```

Comment:
I believe this handler is not used any more. Is this correct? If so let's delete it altogether.